usr/share/rear/layout/prepare/GNU/Linux/110_include_lvm_code.sh

```diff
@@ -140,6 +140,7 @@ EOF
 cat >> "$LAYOUT_CODE" <<EOF
 if [ \$create_volume_group -eq 1 ] ; then
     LogPrint "Creating LVM VG '$vg'; Warning: some properties may not be preserved..."
+    lvm vgremove --force $vg >&2 || true
```

Comment:
In https://github.com/rear/rear/pull/2514#issuecomment-747434614
@gdha suggests `lvm vgremove $vg --force --yes`
i.e. additionally a `--yes` and in
https://github.com/rear/rear/blob/b98687ae9c61084b74d996179a0550abc887c005/usr/share/rear/layout/recreate/default/150_wipe_disks.sh
I had to even use a double force plus yes `--force --force -y`
(but in my case it was `pvremove`) see the comment in my code.